app/data/hpiRepo.ts

```diff
@@ -5,6 +5,7 @@ const getHPIByITL3 = async (itl3: string): Promise<number> => {
     const {
       _avg: { hpi2000: averageHpi },
     } = await prisma.hPI.aggregate({
+      // TODO: Should there be a relationship on ITL3?
```

Comment:
Not quite sure what you mean by this?

Reply:
One to talk through next week and add to the backlog I think. We're currently just string matching on the start/end of an itl3 in multiple tables. 

What we might actually want to model is an ITL3 table, with relationships to other tables. This would be more efficient when it comes to queries and allow us to join multiple tables. Something like this maybe?

```ts
model Itl3 {
  id          Int       @id @default(autoincrement())
  code        String    @unique @db.VarChar(250)
  name        String    @db.VarChar(250)
  
  // Relationships
  GDHI        GDHI[]
  HPI         HPI[]
  Rent        Rent[]
  SocialRentEarnings SocialRentEarnings[]
  GasBills    GasBills[]
  ItlLookup   ItlLookup[]

  @@map("itl3")
}
```

Reply:
I _think_ I see what you mean! Will make a note and we can talk through it at next dev call :)